wheel/generate_type_stubs.py

```diff
@@ -222,7 +222,7 @@ def parse_rust_source(filename: str, upper_case: bool) -> List[Tuple[str, List[s
         "def uncurry(self) -> Tuple[ChiaProgram, ChiaProgram]: ...",
     ],
     "SpendBundle": [
-        "@staticmethod\n    def aggregate(sbs: List[SpendBundle]) -> SpendBundle: ...",
+        "@classmethod\n    def aggregate(sbs: List[SpendBundle]) -> SpendBundle: ...",
```

Comment:
the `aggregate()` function also takes `cls`, right? And it doesn't necessarily return a `SpendBundle`

Reply:
Done